src/ansys/dpf/composites/_indexer.py

```diff
@@ -52,18 +52,52 @@ def setup_index_by_id(scoping: Scoping) -> IndexToId:
     return IndexToId(mapping=indices, max_id=len(indices) - 1)
 
 
-class PropertyFieldIndexerSingleValue(Protocol):
+class PropertyFieldIndexerProtocol(Protocol):
     """Protocol for single value property field indexer."""
 
     def by_id(self, entity_id: int) -> Optional[np.int64]:
```

Comment:
Maybe mention that this function can throw if there is multiple values per entitiy